spec/models/comment_spec.rb

```diff
@@ -0,0 +1,11 @@
+require '../rails_helper'
+
```

Comment:
- [ ] This line should have been written like this `require 'rails_helper'`, kindly make corrections for the tests to pass.